homeassistant/components/xiaomi_miio/select.py

```diff
@@ -34,23 +35,19 @@
 
 
 @dataclass
-class SelectorType:
-    """Class that holds device specific info for a xiaomi aqara or humidifier selectors."""
+class XiaomiMiioSelectDescription(SelectEntityDescription):
+    """A class that describes select entities."""
 
-    name: str = None
-    icon: str = None
-    short_name: str = None
-    options: list = None
-    service: str = None
+    options: tuple = ()
 
 
 SELECTOR_TYPES = {
-    FEATURE_SET_LED_BRIGHTNESS: SelectorType(
-        name="Led brightness",
+    FEATURE_SET_LED_BRIGHTNESS: XiaomiMiioSelectDescription(
+        key=ATTR_LED_BRIGHTNESS,
+        name="Led Brightness",
         icon="mdi:brightness-6",
-        short_name=ATTR_LED_BRIGHTNESS,
-        options=["Bright", "Dim", "Off"],
-        service=SERVICE_SET_LED_BRIGHTNESS,
+        device_class="xiaomi_miio__led_brightness",
```

Comment:
Is the double underscore intentional here, and in case that's a typo, will it cause a breaking change if fixed post-release?

Reply:
Yes, it's intentional, please look [here](https://developers.home-assistant.io/docs/internationalization/core#stringssensorjson).